### PR TITLE
Updated tablets_per_node to include "copying"

### PR DESCRIPTION
**Pull request by na6vj**
Load balancer puts tablets in "Copying" state.
Updated tablets_per_node view to show which nodes have tablets in that state

This satisfies issue#152

### CODE REVIEW

File: tablet-report-parser/moses.pl [hunk 1]
```diff
@@ -805,14 +805,22 @@ sub Create_Views{
     SELECT '$0' as program, '$VERSION' as version, '$opt{STARTTIME_PRINTABLE}' AS run_on, '$opt{LOCALHOST}' as host;
 
   CREATE VIEW tablets_per_node AS 
-    SELECT node_uuid,min(public_ip) as node_ip,min(region ) as region,  count(*) as tablet_count,
-           sum(CASE WHEN private_ip = leader THEN 1 ELSE 0 END) as leaders,
+    SELECT node_uuid,ip as node_ip,zone,  count(*) as tablet_count,
```
Comment (ionthegeek) [inline tablet-report-parser/moses.pl:808]: Why the change from `public_ip` to `ip` here?

Similarly, why the change from `region` to `zone`? One region may have n zones and each zone belongs to one region.

File: tablet-report-parser/moses.pl [hunk 2]
```diff
@@ -805,14 +805,22 @@ sub Create_Views{
...
+           sum(CASE WHEN status='TABLET_DATA_COPYING' THEN 1 ELSE 0 END) as copying,
            sum(CASE WHEN tablet.state = 'TABLET_DATA_TOMBSTONED' THEN 1 ELSE 0 END) as tombstoned,
+           sum(CASE WHEN node_uuid = leader THEN 1 ELSE 0 END) as leaders,
```
Comment (ionthegeek) [inline tablet-report-parser/moses.pl:811]: This changed from `private_ip = leader` to `node_uuid = leader`. Did the definition of `leader` change? If so, where?

File: tablet-report-parser/tablet-report-parser.pl [hunk 3]
```diff
@@ -169,13 +169,22 @@ sub Process_tablet_report{
      SELECT  namespace,table_name, count(*) as total_tablet_count,count(DISTINCT tablet_uuid) as unique_tablet_count, count(DISTINCT node_uuid) as nodes
 	 FROM tablet GROUP BY namespace,table_name;
 CREATE VIEW tablets_per_node AS
-    SELECT node_uuid,min(ip) as node_ip,min(zone) as zone,  count(*) as tablet_count,
+    SELECT node_uuid,ip as node_ip,zone,  count(*) as tablet_count,
```
Comment (ionthegeek) [inline tablet-report-parser/tablet-report-parser.pl:172]: Is `min(ip)` no longer the correct thing to do here? What if there are multiple IPs in this column?

File: tablet-report-parser/tablet-report-parser.pl [hunk 4]
```diff
@@ -169,13 +169,22 @@ sub Process_tablet_report{
...
+        WHERE cluster.type='TSERVER' and cluster.uuid=node_uuid
+        GROUP BY node_uuid, ip, zone 
+    UNION
+    SELECT '~~TOTAL~~',
+	    '*(All '|| (select count(*) from cluster where type='TSERVER') || ' nodes)*', 'ALL',
+       (Select count(*) from tablet),(Select count(*) from tablet WHERE status='TABLET_DATA_COPYING'),
+	   (SELECT count(*) from tablet where state = 'TABLET_DATA_TOMBSTONED'),
+	   (SELECT count(*) from tablet where node_uuid = leader),
+	   (SELECT count(DISTINCT table_name) as table_count from tablet)
+	   ORDER BY 1;
```
Comment (ionthegeek) [inline tablet-report-parser/tablet-report-parser.pl:187]: `ORDER BY` a constant?

**Review by ionthegeek — COMMENTED** [review]
Several comments, mostly requests for clarification.

File: tablet-report-parser/tablet-report-parser.pl (hunk 3 above)
Comment (na6vj) [reply tablet-report-parser/tablet-report-parser.pl:172]: min(ip) is incorrect + not required because the tuple (node_uuid, ip) is unique to each node and both are used in the GROUP BY.

File: tablet-report-parser/tablet-report-parser.pl (hunk 4 above)
Comment (na6vj) [reply tablet-report-parser/tablet-report-parser.pl:187]: The purpose of the "ORDER BY 1" is to get the node_uuid's in order, then the "~TOTAL~" line as the LAST line in the UNION.

File: tablet-report-parser/moses.pl (hunk 1 above)
Comment (na6vj) [reply tablet-report-parser/moses.pl:808]: Thanks for catching this - this was a mistake - it is being reverted, and corrected.

File: tablet-report-parser/moses.pl (hunk 2 above)
Comment (na6vj) [reply tablet-report-parser/moses.pl:811]: Thanks for catching this - this was a mistake - it is being reverted, and corrected.

**Comment by na6vj** [comment]
Previous comments have been resolved with new checkins.

**Review by ionthegeek — APPROVED** [review]
